doc/mgr/prometheus.rst

```diff
@@ -29,6 +29,24 @@ configurable with ``ceph config-key set``, with keys
 ``mgr/prometheus/server_addr`` and ``mgr/prometheus/server_port``.
 This port is registered with Prometheus's `registry <https://github.com/prometheus/prometheus/wiki/Default-port-allocations>`_.
 
+RBD IO statistics
+-----------------
+
+The module allows to collect per RBD image IO statistics by enabling
+dynamic OSD performance counters. The statistics is gathered for the
```

Comment:
"The statistics are gathered for all images in the pools that are specified in the ..."